### ISSUE TITLE
Run setup.py -x flags in scripts/update

### ISSUE THREAD

**Issue by jisantuc** [PublicMapping/districtbuilder-classic#474]
Once issues from #472 are resolved, create a `scripts/bootstrap` that gets the application into a working state

**Comment by hectcastro** [comment]
Could we tuck whatever is destined for `bootstrap` into `update`? That seems to be the convention across most projects (RF is an outlier because it was being setup while things were still being figured out, and is slowly being migrated).

**Comment by jisantuc** [comment]
Yeah that's fine, I'll update the issue title accordingly. One less script to run :+1:
